### PR TITLE
dev-lang/mono: Multilib support

### PR DESCRIPTION
**Pull request by msva**

### CODE REVIEW

File: dev-lang/mono/mono-4.4.1.0.ebuild [hunk 1]
```diff
@@ -85,31 +90,34 @@ src_prepare() {
...
+	multilib_is_native_abi || big_arr="disable"
+
 	local myeconfargs=(
 		--disable-silent-rules
 		$(use_with xen xen_opt)
 		--without-ikvm-native
 		--disable-dtrace
 		$(use_with doc mcs-docs)
 		$(use_enable nls)
+		--${big_arr}-big-arrays
```
Comment (mgorny) [inline dev-lang/mono/mono-4.4.1.0.ebuild]: Err, wtf? Are you doing some poor assumptions at what native and non-native ABI will be? :-P

Comment (msva) [inline dev-lang/mono/mono-4.4.1.0.ebuild]: Actually, I do (ok, actually, it is what guys on `#gentoo-dev-help` suggested), but anyway I never heard people on <abiname>64 had something other than <sameabi>32 in multilib.

// Although, I just realized (when finished previous line) that multilib_is_native_abi will be true on native x86 :D.

Comment (floppym) [inline dev-lang/mono/mono-4.4.1.0.ebuild]: Enabling this seems like a bad idea.

```
--enable-big-arrays - Enable use of arrays with indexes larger than Int32.MaxValue.

By default Mono has the same limitation as .NET on Win32 and Win64 and limits array indexes to 32-bit values (even on 64-bit systems).

In certain scenarios where large arrays are required, you can pass this flag and Mono will be built to support 64-bit arrays.

This is not the default as it breaks the C embedding ABI that we have exposed through the Mono development cycle.
```

Comment (floppym) [inline dev-lang/mono/mono-4.4.1.0.ebuild]: If you really want to add it, it belongs behind a separate USE flag.

Comment (msva) [inline dev-lang/mono/mono-4.4.1.0.ebuild]: Yeah, I rethink that decision and pushed bump without it :)

**Comment by msva** [comment]
Eerm... Why do it closed? O_o

**Pull request closed without merging**